### ISSUE TITLE
Model: Peetoner NutriVision

### ISSUE THREAD

**Issue by robertbigk** [tuya/tuya-home-assistant#24]
https://www.petoneer.com/NutriVision

Is detect and add for IOT and smart live but bo way add option to Home Asssistant

{
  "result": {
    "active_time": 1624556244,
    "biz_type": 18,
    "category": "sp",
    "create_time": 1624556244,
    "icon": "smart/icon/ay1541381385946hoIoV/156360121030885b7eb80.png",
    "id": "bf18e3e146dd0a86f34prj",
    "ip": 
    "lat": "52.0794936",
    "local_key":
    "lon": "21.0107744",
    "name": "Nutri Vision",
    "online": true,
    "owner_id": "18127768",
    "product_id": "bmwdr0lrdhyqwktb",
    "product_name": "Nutri Vision",
    "status": [
      {
        "code": "basic_indicator",
        "value": false
      },
      {
        "code": "basic_flip",
        "value": false
      },
      {
        "code": "basic_osd",
        "value": false
      },
      {
        "code": "basic_nightvision",
        "value": "0"
      },
      {
        "code": "movement_detect_pic",
        "value": "$"
      }
    ],
    "sub": false,
    "time_zone": "+02:00",
    "uid": "eu1604020388098rm3hC",
    "update_time": 1624556268,
    "uuid": "tkcx733561be91fc221b"
  },
  "success": true,
  "t": 1624557400758
}

**Comment by tsutsuku** [comment]
@robertbigk 
The Smart Camera which's category is sp is not supported now.
We will consider adding it in the future.

**Comment by robertbigk** [comment]
Ok sorry for my ash but in the future is what time weeks months years? Ok but possible to add for now a basic switch on/off for food distribution and camera another time?

**Comment by jkottier** [comment]
Indeed, basic (doorbell) camera support with at least movement and doorbell pressed notifications would be highly appreciated. I think a lot of Home Assistant / Tuya users are waiting for this to become available.

**Comment by robertbigk** [comment]
I don't have a lot knowledge i try to learn much I can if I can help just write me if need. If need to testing device just write me:)

**Comment by robertbigk** [comment]
When You need log for this device let me know.

**Comment by robertbigk** [comment]
Hi any hope for my device because is get lower and lower on the list:)

**Comment by hawkeye1103** [comment]
I subscribe to this enhancement. Would like to see it fully implemented in home assistant. Thx

**Comment by hawkeye1103** [comment]
{
  "result": {
    "properties": [
      {
        "code": "basic_indicator",
        "custom_name": "",
        "dp_id": 101,
        "time": 1709324850973,
        "value": false
      },
      {
        "code": "basic_flip",
        "custom_name": "",
        "dp_id": 103,
        "time": 1709321925169,
        "value": false
      },
      {
        "code": "basic_osd",
        "custom_name": "",
        "dp_id": 104,
        "time": 1709321925386,
        "value": false
      },
      {
        "code": "basic_nightvision",
        "custom_name": "",
        "dp_id": 108,
        "time": 1709324851142,
        "value": "0"
      },
      {
        "code": "express_feed",
        "custom_name": "",
        "dp_id": 201,
        "time": 1709309820359,
        "value": 0
      },
      {
        "code": "food_weight",
        "custom_name": "",
        "dp_id": 202,
        "time": 1709309835062,
        "value": 1
      },
      {
        "code": "control",
        "custom_name": "",
        "dp_id": 203,
        "time": 1709309835375,
        "value": 1
      },
      {
        "code": "realtime_data",
        "custom_name": "",
        "dp_id": 204,
        "time": 1709310037632,
        "value": 0
      },
      {
        "code": "weight",
        "custom_name": "",
        "dp_id": 205,
        "time": 1709309835825,
        "value": 101
      },
      {
        "code": "history_data",
        "custom_name": "",
        "dp_id": 206,
        "time": 1709322194406,
        "value": 259
      },
      {
        "code": "schedule",
        "custom_name": "",
        "dp_id": 207,
        "time": 1709309836358,
        "value": ""
      },
      {
        "code": "feed_beep",
        "custom_name": "",
        "dp_id": 231,
        "time": 1709324851363,
        "value": false
      },
      {
        "code": "feed_voice_record",
        "custom_name": "",
        "dp_id": 238,
        "time": 1709309820359,
        "value": 1
      },
      {
        "code": "power_mode",
        "custom_name": "",
        "dp_id": 239,
        "time": 1709309836878,
        "value": 0
      }
    ]
  },
  "success": true,
  "t": 1709325232833,
  "tid": "04908387d80b11eeb8e44adb5c011a40"
}
